ci/tests/ehcache3/run-tests-ehcache3.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+source ./ci/functions.sh
+
+runBuild=false
+echo "Reviewing changes that might affect the Gradle build..."
+currentChangeSetAffectsTests
+retval=$?
+if [ "$retval" == 0 ]
+then
+    echo "Found changes that require the build to run test cases."
+    runBuild=true
+else
+    echo "Changes do NOT affect project test cases."
+    runBuild=false
+fi
+
+if [ "$runBuild" = false ]; then
+    exit 0
+fi
+
+
+gradle="./gradlew $@"
+gradleBuild=""
+gradleBuildOptions="--build-cache --configure-on-demand --no-daemon -DtestCategoryType=EHCACHE3"
```

Comment:
Change tag to EHCACHE in general; safety against future upgrades and maintenance of ehcache-based work.